src/Destructurama.FSharp/Destructurama.FSharp.fsproj

```diff
@@ -1,52 +1,18 @@
 <Project Sdk="Microsoft.NET.Sdk">
-  <PropertyGroup>
-    <!--
-      We mimic the same frameworks as Serilog as much as we can.
-      FSharp.Core itself has a minimum bound of netstandard 1.6 for now, so that pins us.
-     -->
-    <TargetFrameworks>netstandard1.6;netstandard2.0</TargetFrameworks>
-    <DisableImplicitFSharpCoreReference>true</DisableImplicitFSharpCoreReference>
-    <DisableImplicitSystemValueTupleReference>true</DisableImplicitSystemValueTupleReference>
-    <WarningLevel>5</WarningLevel>
-    <GenerateDoc>true</GenerateDoc>
 
-    <!-- Debug/Symbol properties -->
-    <DebugType>portable</DebugType>
-    <PublishRepositoryUrl>true</PublishRepositoryUrl>
-    <!-- Explicitly allowing PDBs in the main package because it's much nicer for consumers/tooling and this is a small library. -->
-    <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
-    <IncludeSymbols>true</IncludeSymbols>
-    <SymbolPackageFormat>snupkg</SymbolPackageFormat>
-    
-    <!-- Nuget Package properties -->
-    <IsPackable>true</IsPackable>
-    <PackageId>Destructurama.FSharp</PackageId>
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
     <Title>F# Destructuring extensions for Serilog</Title>
-    <Authors>Destructurama Contributors</Authors>
-    <PackageDescription>Native support for destructuring F# types when logging to Serilog.</PackageDescription>
-    <Description>
-      A destructuring policy for Serilog loggers that understands how to unwrap and display common F# data types, such as Tuples and Unions.
-    </Description>
-    <Copyright>Copyright 2019 Destructurama Contributors, Serilog Contributors</Copyright>
```

Comment:
did this go somewhere else? serilog etc has this
IIRC authors needs to stay too?